### PR TITLE
Strimzi proposal for CNCF Incubation

### PR DESCRIPTION
**Pull request by ppatierno**
Strimzi has been in CNCF Sandbox since September 2019. 
The project and community has shown continuous growth year after year.
We would like to propose Strimzi for Incubation.

### CODE REVIEW

**Comment by dims** [comment]
Folks, @erinaboyd will be the TOC sponsor for Strimzi. thanks Erin!

**Comment by ppatierno** [comment]
@erinaboyd hi! Is there anything we can help with in order to start the process? Just let us know, anything you need from us. Thanks! FYI @scholzj @tombentley

**Comment by ppatierno** [comment]
@amye @dims now that @erinaboyd is not a TOC member anymore, how will this process proceed? Do we have to do something to move forward on our side, or waiting for a TOC member to be assigned, something else?

**Comment by mattfarina** [comment]
@ppatierno I'll be reaching out but I want to post that I'm the TOC member who will be pushing this forward.

**Comment by ppatierno** [comment]
@mattfarina thanks! That's a great news to see this finally moving forward. Let us know whatever you need from us. FYI @scholzj @tombentley

File: proposals/incubation/strimzi.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+* Kubernetes
+* [Prometheus][prom] for metrics and monitoring
+* [Open Policy Agent][opa] for authorization
+* [KEDA][keda] for autoscaling of Kafka clients
+* [Jaeger][jaeger] and [OpenTelemetry][otel] for tracing
+
+In addition, Strimzi is used by [Knative][knative] to provide its Kafka source.
+
+Strimzi can be installed using [Helm][helm] charts or using [OperatorHub.io][operatorhub], which is part of the [Operator Framework][operatorframework] project.
```
Comment (mattfarina) [inline proposals/incubation/strimzi.md:22]: If I remember correctly, OperatorHub is NOT part of the Operator Framework project. When the Operator Framework was given to the CNCF, OperatorHub was not part of it and has it's own GitHub org. I think this needs to be updated.

**Review by mattfarina — COMMENTED** [review]
I have one nit for the initial PR. Note, I've not reviewed it for content and structure, yet.

File: proposals/incubation/strimzi.md (hunk 1 above)
Comment (scholzj) [reply proposals/incubation/strimzi.md:22]: I guess we can maybe rephrase it something like this?

```suggestion
Strimzi can be installed using [Helm][helm] charts or using Operator Lifecycle Manager, which is part of the [Operator Framework][operatorframework] project.
Both the Helm charts and OLM bundles are available on [Artifact Hub](https://artifacthub.io/).
```

The Operator Lifecycle Manager seems to be part of Operator Framework (https://github.com/operator-framework/operator-lifecycle-manager) -> just the Operator directory isn't. But the artifacts are listed also in Artifact Hub that is part of CNCF if I remember correctly.

Comment (ppatierno) [reply proposals/incubation/strimzi.md:22]: I agree with Jakub's proposal. @mattfarina let me know what do you think about it, and I will update the PR. Thanks!

Comment (ppatierno) [reply proposals/incubation/strimzi.md]: I pushed the suggested change, let me know if that's not ok and we can re-open the conversation.

**Comment by mattfarina** [comment]
For reference, the [Due Diligence can be found here](https://docs.google.com/document/d/1o7vhzUQhAFm8sYz6aV_JimRXyKIQR_ZYb0OXpmaMtug/edit).

**Comment by mobs75** [comment]
👍
